backend/build.sh

```diff
@@ -11,8 +11,8 @@ SCOPE_SRC=$GOPATH/src/github.com/weaveworks/scope
 # will have awkward ownership.  So we switch to a user with the
 # same user and group IDs as source directory.  We have to set a
 # few things up so that sudo works without complaining later on.
-uid=$(stat --format="%u" $SCOPE_SRC)
-gid=$(stat --format="%g" $SCOPE_SRC)
+uid=$(stat -c "%u" $SCOPE_SRC)
```

Comment:
stat might have a different parameters whether it is the real "stat" or the "stat" from busybox.